### ISSUE TITLE
Result based validation

### ISSUE THREAD

**Issue by rdewijn** [tercen/vsn_shiny_operator#6]
Quite some extra code was added to manage the views from Tercen etc. Can we check if the results are equal to that of teh BN version of this operator?

**Comment by ginberg** [comment]
@rdewijn I did check the results for the previous version (vsn) since I have got that version in my Bionavigator. But I have an issue installing the shinyvsn 1.8; it's saying: Failed to install package shinyvsn: required version 1.8 installed version 1.5
Do you know how I can fix this? I asked Faris and Alex but they couldn't help me with it

**Comment by rdewijn** [comment]
You may have to use  a newer version of R with BN for this one (e.g. R-4.0.3)

**Comment by ginberg** [comment]
Yes, I got it working with R 4.0.3. I validated the results are correct. I added the output from Bionavigator to the [Tercen project ](https://tercen.com/pamgene/p/46842dc092dac0594fe844174f6a8e8c), for 2 cases: one with and one without reference data. Tis operator is ready for code review again, the other issues are also fixed
